tests/lib/snaps/test-snapd-daemon-notify/meta/hooks/install

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
```

Comment:
Nitpick: this could be a plain shell script.

Reply:
Done